src/Explorer/Panes/SettingsPaneR/index.tsx

```diff
@@ -0,0 +1,433 @@
+    );
+
+    close();
+  };
+
+  const isCustomPageOptionSelected = () => {
+    return pageOption === Constants.Queries.CustomPageOption;
+  };
+
+  const isUnlimitedPageOptionSelected = () => {
+    return pageOption === Constants.Queries.UnlimitedPageOption;
+  };
+
+  const onUnlimitedPageOptionKeyDown = (event: KeyboardEvent) => {
+    if (event.keyCode === Constants.KeyCodes.Enter || event.keyCode === Constants.KeyCodes.Space) {
+      setPageOption(Constants.Queries.UnlimitedPageOption);
+      event.stopPropagation();
+    }
+  };
+
+  const onCustomPageOptionsKeyDown = (event: KeyboardEvent) => {
+    if (event.keyCode === Constants.KeyCodes.Enter || event.keyCode === Constants.KeyCodes.Space) {
+      setPageOption(Constants.Queries.CustomPageOption);
+      event.stopPropagation();
+    }
+  };
+
+  const onJsonDisplayResultsKeyDown = (event: KeyboardEvent) => {
+    if (event.keyCode === Constants.KeyCodes.Enter || event.keyCode === Constants.KeyCodes.Space) {
+      setGraphAutoVizDisabled("true");
+      event.stopPropagation();
+    }
+  };
+
+  const onGraphDisplayResultsKeyDown = (event: KeyboardEvent) => {
+    if (event.keyCode === Constants.KeyCodes.Enter || event.keyCode === Constants.KeyCodes.Space) {
+      setGraphAutoVizDisabled("false");
+      event.stopPropagation();
+    }
+  };
+
+  const _loadSettings = () => {
+    setIsExecuting(true);
+    try {
+      setPageOption(
+        LocalStorageUtility.getEntryNumber(StorageKey.ActualItemPerPage) === Constants.Queries.unlimitedItemsPerPage
+          ? Constants.Queries.UnlimitedPageOption
+          : Constants.Queries.CustomPageOption
+      );
+      setCustomItemPerPage(LocalStorageUtility.getEntryNumber(StorageKey.CustomItemPerPage));
+    } catch (exception) {
+      setFormErrors("Unable to load your settings");
+      setFormErrorsDetails(exception);
+    } finally {
+      setIsExecuting(false);
+    }
+  };
+
+  return (
+    <div data-bind="visible: visible, event: { keydown: onPaneKeyDown }">
+      <div className="contextual-pane-out" data-bind="click: cancel, clickBubble: false" />
+      <div className="contextual-pane" id="settingspane">
+        {/* Settings Confirmation form - Start */}
+        <div className="contextual-pane-in">
+          <form className="paneContentContainer" onSubmit={submit}>
+            {/* Settings Confirmation header - Start */}
+            <div className="firstdivbg headerline">
+              <span role="heading" aria-level={2}>
+                Settings
+              </span>
+              <div className="closeImg" role="button" aria-label="Close pane" tabIndex={0} onClick={closePanel}>
+                <img src="../../../images/close-black.svg" title="Close" alt="Close" />
+              </div>
+            </div>
+            {/* Settings Confirmation header - End */}
+            {/* Settings Confirmation errors - Start */}
```

Comment:
Remove any comments like this. They are old and not needed.

Reply:
done